### ISSUE TITLE
Upgrade to Alpine Linux 3.16.1 for fixing the busybox (CVE-2022-30065)

### ISSUE THREAD

**Issue by vienleidl** [dotnet/dotnet-docker#3957]
Alpine Linux has released a security update for busybox to fix the vulnerabilities. The affected package versions are prior to 1.35.0-r17 [(CVE-2022-30065)](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-30065). The GHSA link is [here](https://github.com/advisories/GHSA-gq73-rh3m-3php) for more information.

The [busybox CVE-2022-3065](https://git.alpinelinux.org/aports/commit/?h=v3.16.1&id=e58f0e341603e9facc1e675722367174b45e0d0c) was fixed on Alpine Linux with Releases: [3.16.1](https://www.alpinelinux.org/posts/Alpine-3.16.1-released.html), [3.13.11, 3.14.7 AND 3.15.5](https://www.alpinelinux.org/posts/Alpine-3.13.11-3.14.7-3.15.5-released.html).

**Comment by mthalman** [comment]
When reporting vulnerabilities issues, please always specify the image tag that's affected.

In this case, I'm looking at `mcr.microsoft.com/dotnet/aspnet:alpine3.16`. It is up to date with what is provided by the Alpine base image tag (`alpine:3.16`).

```
C:\> docker pull mcr.microsoft.com/dotnet/aspnet:6.0-alpine3.16
6.0-alpine3.16: Pulling from dotnet/aspnet
530afca65e2e: Already exists
6207422b9602: Already exists
97f3adc41521: Already exists
5b011df4216c: Already exists
Digest: sha256:28d95bfe423e5db1d050eff6177dd816a8afa2a161697194fb64310940db6cb0
Status: Downloaded newer image for mcr.microsoft.com/dotnet/aspnet:6.0-alpine3.16
mcr.microsoft.com/dotnet/aspnet:6.0-alpine3.16

C:\> docker run --rm mcr.microsoft.com/dotnet/aspnet:6.0-alpine3.16 cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.16.1
PRETTY_NAME="Alpine Linux v3.16"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
```

Alpine hasn't provided an updated version of busybox in their base image, which is where busybox is installed by default. There is already a discussion of this at https://github.com/alpinelinux/docker-alpine/issues/264. For packages that .NET doesn't install, we defer to the Alpine maintainers on the issue of determining when container images should have packages upgraded.

**Comment by vienleidl** [comment]
> C:\> docker pull mcr.microsoft.com/dotnet/aspnet:6.0-alpine3.16

Hi @mthalman , sorry I forgot, I use this image mcr.microsoft.com/dotnet/sdk:6.0-alpine

**Comment by mthalman** [comment]
> > C:> docker pull mcr.microsoft.com/dotnet/aspnet:6.0-alpine3.16
> 
> Hi @mthalman , sorry I forgot, I use this image mcr.microsoft.com/dotnet/sdk:6.0-alpine

Got it. Same comment from [above](https://github.com/dotnet/dotnet-docker/issues/3957#issuecomment-1201205659) applies to that image.

**Comment by vienleidl** [comment]
It's fixed in the new release of Alpine Linux 3.16.2. I've just checked again, the security finding is gone.
```
docker run --rm mcr.microsoft.com/dotnet/sdk:6.0-alpine cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.16.2
PRETTY_NAME="Alpine Linux v3.16"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
```

```
docker run --rm mcr.microsoft.com/dotnet/sdk:6.0-alpine apk busybox info
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/main: No such file or directory
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.16/community: No such file or directory
busybox-1.35.0-r17 description:
Size optimized toolbox of many common UNIX utilities

busybox-1.35.0-r17 webpage:
https://busybox.net/

busybox-1.35.0-r17 installed size:
940 KiB
```

**Comment by mthalman** [comment]
> It's fixed in the new release of [Alpine Linux 3.16.2](https://alpinelinux.org/posts/Alpine-3.13.12-3.14.8-3.15.6-3.16.2-released.html).

Yep. And .NET images were rebuilt yesterday as a result of that update. So things should be all good now.
